ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -536,8 +546,8 @@ public void testAccountEqual() throws Exception {
     assertTrue("Two accounts should be equal.", accountNoContainer.equals(accountNoContainerDuplicate));
 
     // Check two accounts with same fields and containers.
-    Account accountWithContainers = Account.fromJson(accountJsonLike);
-    Account accountWithContainersDuplicate = Account.fromJson(accountJsonLike);
+    Account accountWithContainers = Account.fromJson(refAccountJson);
```

Comment:
for testUnknownAccountAndContainer(), can we test for new fields too

Reply:
added